src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
```

Comment:
Extending the 'ColorSupplier' class is not the best practice here since 'Lottery' does not have an 'is-a' relationship with 'ColorSupplier'. Prefer composition over inheritance when there is no logical 'is-a' relationship.